testutil/configurator/configurator.go

```diff
@@ -0,0 +1,109 @@
+package configurator
+
+import (
+	runtimev1alpha1 "cosmossdk.io/api/cosmos/app/runtime/v1alpha1"
+	appv1alpha1 "cosmossdk.io/api/cosmos/app/v1alpha1"
+	authmodulev1 "cosmossdk.io/api/cosmos/auth/module/v1"
+	bankmodulev1 "cosmossdk.io/api/cosmos/bank/module/v1"
+	paramsmodulev1 "cosmossdk.io/api/cosmos/params/module/v1"
+	"cosmossdk.io/core/appconfig"
+	"cosmossdk.io/depinject"
+)
+
+var beginBlockOrder = []string{
```

Comment:
Note: I am using naked strings here to avoid a hard dependency on modules on the configurator package.